src/js/controllers/tab-home.js

```diff
@@ -32,8 +32,10 @@ angular.module('copayApp.controllers').controller('tabHomeController',
             $log.warn(err);
             return;
           }
-
-          if (newRelease) $scope.newRelease = true;
+          if (newRelease) {
+            $scope.newRelease = true;
+            $scope.updateText = gettextCatalog.getString('There is a new version of ') + $scope.name + gettextCatalog.getString(' available');
```

Comment:
It 's not clear for translators. Should use interpolation instead: https://angular-gettext.rocketeer.be/dev-guide/api/angular-gettext/